conda/plugins/manager.py

```diff
@@ -184,12 +191,7 @@ def get_solver_backend(self, name: str | None = None) -> type[Solver]:
             name = context.solver
         name = name.lower()
 
-        # Build a mapping between a lower cased backend name and
-        # solver backend class provided by the installed plugins.
-        solvers_mapping = {
-            solver.name.lower(): solver.backend
-            for solver in self.get_hook_results("solvers")
-        }
+        solvers_mapping = self.get_solvers()
 
         # Look up the solver mapping and fail loudly if it can't
```

Comment:
It's worth including the error handling in the getter

Reply:
what error handling are you referring to?